scripts/runeofecdf2nc.py

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
```

Comment:
I assume you based this off of the other run scripts. I haven't looked closely but they look good to me.

Reply:
yes. I ran them through black code formatter, and they were left unchanged.